packages/@react-types/menu/src/index.d.ts

```diff
@@ -56,3 +56,13 @@ export interface MenuProps<T> extends CollectionBase<T>, MultipleSelection {
 
 export interface AriaMenuProps<T> extends MenuProps<T>, DOMProps, AriaLabelingProps {}
 export interface SpectrumMenuProps<T> extends AriaMenuProps<T>, StyleProps {}
+
+export interface SpectrumActionMenuProps<T> extends CollectionBase<T>, DOMProps, AriaLabelingProps {
```

Comment:
Should there be two ids? one for the button and one for the menu? in which case does DOMProps make sense?

Reply:
IMO it should go on the button

Reply:
if I get it right you don't want to have 2 ids in the API - one thing we could do is to automatically assign an ID to the menu from the button id? Let's say the button ID is `my-button` then we'd assign `my-button-actionmenu-menu` do other controls do something like that already?